website/addons/dataverse/model.py

```diff
@@ -71,6 +72,24 @@ def dataset_id(self, value):
     def complete(self):
         return bool(self.has_auth and self.dataset_doi is not None)
 
+    @property
+    def folder_id(self):
+        return self.dataset_id
+
+    @property
+    def folder_path(self):
+        pass
+
+    @property
+    def nodelogger(self):
```

Comment:
Is this being used to create dataverse logs? If not, you might want to add a note to the PR description to do that in a future PR.